app/controllers/concerns/test_track/controller.rb

```diff
@@ -9,8 +9,20 @@ module TestTrack::Controller
     around_action :manage_test_track_session
   end
 
+  class_methods do
+    def require_feature_flag(feature_flag, *args)
+      before_action(*args) do
+        not_found unless test_track_visitor.ab(feature_flag, context: self.class.name.demodulize.underscore)
+      end
+    end
+  end
+
   private
 
+  def not_found
```

Comment:
This is pretty a global name for a library mixin - we might conflict with an app developer’s different method. I’d probably roll this line into the callback